src/models/group.js

```diff
@@ -1,21 +1,24 @@
 import mongoose from 'mongoose';
+import uuid from 'uuid';
 import User from './user';
 
 const modelName = 'Group';
+const popAll = 'users creator invitedUsers';
```

Comment:
Now that I think of it, it might be easier just to make this a static method on the group schema, that way we won't have to do an extra export.import and we know what the `populateAll()` function relates to.

Reply:
Wouldn't this instead be an **instance** method that is called on any `Group` document that we want to populate?

Reply:
Yeah my bad